astropy/coordinates/angles/core.py

```diff
@@ -11,11 +11,12 @@
 import numpy as np
 
 from astropy import units as u
+from astropy.units.quantity import SpecificTypeQuantity
 from astropy.utils import isiterable
 
-from . import angle_formats as form
+from . import formats
 
-__all__ = ["Angle", "Latitude", "Longitude"]
+__all__ = []
```

Comment:
Let's leave this change until the APE discussion has reached a conclusion -- as few changes as possible in renaming exercises like these!

Reply:
✅ done.